packages/core/src/signer/ckb/signerCkbMultisig.ts

```diff
@@ -0,0 +1,245 @@
+import { Address } from "../../address/index.js";
+import {
+  multisigMetadataFromPubkeys,
+  Script,
+  SinceLike,
+  Transaction,
+  TransactionLike,
+} from "../../ckb/index.js";
+import { CellDepInfo, Client, KnownScript } from "../../client/index.js";
+import { Hex, hexConcat, hexFrom, HexLike } from "../../hex/index.js";
+import { SignerCkbPrivateKey } from "./signerCkbPrivateKey.js";
+
+export interface MultisigInfo {
+  pubkeys: HexLike[];
+  threshold: number;
+  mustMatch: number;
+  since?: SinceLike;
+  multisigScript?:
```

Comment:
Is it possible to let the `Signer` support both script versions at the same time? e.g., we can let the `Signer` return two addresses for `getAddresses`. Similar to https://github.com/ckb-devrel/ccc/blob/master/packages/core/src/signer/evm/signerEvm.ts